redundant-bmc/src/sibling_impl.hpp

```diff
@@ -168,6 +168,21 @@ class SiblingImpl : public Sibling
         return std::nullopt;
     }
 
+    /**
+     * @brief Returns if the sibling has a failover in progress
+     *
+     * @return - If in progress, or nullopt if not available
+     */
+    std::optional<bool> getFailoverInProgress() const override
+    {
+        if (getInterfacePresent() && hasHeartbeat())
+        {
+            return redundancy.failoverInProgress;
```

Comment:
I'm missing something here, how do we actually get the info from the sibling on whether it had the fail over in progress file? Is it something in the op-proc-control CFAM interface?